src/main/java/core/basesyntax/Zoo.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+
+public class Zoo {
+
+    public static void main(String[] args) {
+        ArrayList<Mammal> cage = new ArrayList<>();
+        ArrayList<Fish> aquarium = new ArrayList<>();
+        ArrayList<Bird> birdCage = new ArrayList<>();
```

Comment:
let's use our knowledge of generics and wildcard here
so that we won't need casting: `cage.add((Mammal) animal);`